From 0639fcde3914ad94671c2afe6f1e0b819a702dff Mon Sep 17 00:00:00 2001
From: Guillem Jover <guillem@debian.org>
Date: Tue, 4 Aug 2020 20:51:44 +0200
Subject: [PATCH] aptitude-changelog-parser: Quiesce Dpkg::Changelog parse
 warnings

The Dpkg::Changelog object gets initialized with verbose mode set to 1,
so all parsing warnings get printed on stderr, which messes up the
output when showing the changelogs from the TUI.

Setting this option to 0 before the call allows one to disable the
parse_error warnings from Dpkg::Changelog, which we do not want for
aptitude-changelog-parser.

Closes: #967911
---
 aptitude-changelog-parser | 1 +
 1 file changed, 1 insertion(+)

diff --git a/aptitude-changelog-parser b/aptitude-changelog-parser
index aee1dfd1..fe5cd9ff 100755
--- a/aptitude-changelog-parser
+++ b/aptitude-changelog-parser
@@ -15,6 +15,7 @@ eval {
 # Usage: aptitude-changelog-parser [<filename> [<from-version>]]
 
 my %opts;
+$opts{verbose} = 0;
 if (scalar @ARGV >= 1) {
     $opts{file} = shift @ARGV;
 }
-- 
2.40.1

